### PR TITLE
Improving install-guide

### PR DESCRIPTION
**Pull request by anpaz-msft**
Creatd a VS section that explains how to install the QDK in VS directly
Changed the links of the first notebooks to point to binder now that that is working.
Point notebooks links to binder.
Couple of other minor changes.

### CODE REVIEW

File: articles/install-guide/jupyter.md [hunk 1]
```diff
@@ -33,7 +39,7 @@ dotnet iqsharp install
 
 ## Usage ##
 
-Once installed, start jupyter notebook from the command line using:
+Once the `Q#` Jupyter kernel is installed, start jupyter notebook from the command line using:
 
 ```Command Prompt
 jupyter notebook
```
Comment (tcNickolas) [inline articles/install-guide/jupyter.md]: GitHub doesn't let me leave this comment in a proper place: should the link on line 51 point to Binder for an interactive version of this notebook?

File: articles/install-guide/python.md [hunk 2]
```diff
@@ -13,25 +13,31 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
 
 ## Pre-requisites ##
 
-To get started with the `qsharp` package, you'll need the following prerequisites:
+Before installing the `qsharp` package, you'll need the following pre-requisites:
 - IQ#.
```
Comment (tcNickolas) [inline articles/install-guide/python.md]:
```suggestion
- IQ#,
```

File: articles/install-guide/python.md [hunk 3]
```diff
@@ -13,25 +13,31 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
...
 - Python 3.6 or later,
```
Comment (tcNickolas) [inline articles/install-guide/python.md]:
```suggestion
- Python 3.6 or later.
```

File: articles/install-guide/python.md [hunk 4]
```diff
@@ -13,25 +13,31 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
...
 
-Make sure you have followed the installation steps in the [getting started guide](xref:microsoft.quantum.install) for IQ#.
+IQ# provides the core functionality of compiling and simulating Q# operations.
```
Comment (tcNickolas) [inline articles/install-guide/python.md]: I don't have the context for this change, but it looks a bit weird - we point people here from install guide, and this part of the instructions is duplicated. Should we somehow point it out? (+ the same comment about 10 minutes applies)

File: articles/install-guide/python.md [hunk 5]
```diff
@@ -13,25 +13,31 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
...
 
-Install `qsharp` using `pip`:
+Once Python and IQ# are installed, install the `qsharp` package using `pip`:
 
 ```
 pip install qsharp
 ```
 
-You will also need to explicitly install the `iqsharp` kernel using this command:
+You will also need to explicitly install the `Q#` kernel using this command:
```
Comment (tcNickolas) [inline articles/install-guide/python.md]:
```suggestion
You will also need to explicitly install the Q# kernel using this command:
```

File: articles/install-guide/csharp.md [hunk 6]
```diff
@@ -20,19 +20,19 @@ To write C# programs, download and install the **.NET Core SDK** 2.1 or later fr
 ## Installation ##
```
Comment (tcNickolas) [inline articles/install-guide/csharp.md]: Comment for pre-requisites: the user doesn't need to install .net core here if they have already installed Q#, so I would remove this section.

File: articles/install-guide/csharp.md [hunk 7]
```diff
@@ -20,19 +20,19 @@ To write C# programs, download and install the **.NET Core SDK** 2.1 or later fr
...
 
 To download the latest templates for creating new C# projects and libraries that invoke Q# operations, execute this from a command line:
```
Comment (tcNickolas) [inline articles/install-guide/csharp.md]: This part needs a clarification that you need to do it only if you've installed from command line, if you're doing it in VS you're done.

File: articles/install-guide/index.md [hunk 8]
```diff
@@ -9,54 +9,97 @@ uid: microsoft.quantum.install
...
-Language kernel: 0.5.1903.2501
+Language kernel: 0.5.1903.2902
 Jupyter core: 1.1.12077.0
 ```
 
+
+### Visual Studio and VS Code extensions ###
+
+The Quantum Development Kit offers tight integration with [Visual Studio](https://visualstudio.microsoft.com/vs/) 
+and [Visual Studio Code](https://code.visualstudio.com/) that improve the development experience.
```
[the remark this answers is not in the document]
Comment (MikeDodaro) [reply articles/install-guide/index.md]: It looks like lines 50-67 are redundant to the text beginning at line 70.

**Comment by anpaz-msft** [comment]
This has been super-seeded by other changes.

**Pull request closed without merging**